scripts/bump-release

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+#
+# Attempt to bump the docker2aci release to the specified version by replacing
+# all occurrences of the current/previous version.
+#
+# Generates two commits: the release itself and the bump to the next +git
+# version
+#
+# YMMV, no disclaimer or warranty, etc.
+
+# make sure we are running in a toplevel directory
+if ! [[ "$0" =~ "scripts/bump-release" ]]; then
+	echo "This script must be run in a toplevel docker2aci directory"
+	exit 255
+fi
+
+if ! [[ "$1" =~ ^v[[:digit:]]+.[[:digit:]]+.[[:digit:]]$ ]]; then
```

Comment:
The dots have to be escaped, otherwise running `./scripts/bump-release v0X9Y1` does not bail out. I guess this needs to be fixed in rkt too.